ios/COVIDSafePaths.xcodeproj/project.pbxproj

```diff
@@ -403,7 +400,7 @@
 				C53FD09724719AD1006D3268 /* CoreGraphics.framework in Frameworks */,
 				C53FD09824719AD1006D3268 /* CoreData.framework in Frameworks */,
 				C53FD09924719AD1006D3268 /* Accelerate.framework in Frameworks */,
-				68EFB21324856EC6003D84F3 /* Scrypt in Frameworks */,
+				68EFB21324856EC6003D84F3 /* (null) in Frameworks */,
```

Comment:
these (null) in x are suspicious

Reply:
they are! working through getting it all in order.